packages/mask/src/plugins/Game/SNSAdaptor/WalletConnectDialog.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react'
+import { useRemoteControlledDialog } from '@masknet/shared-base-ui'
+import DialogContent from '@mui/material/DialogContent'
+import { PluginGameMessages } from '../messages'
+
+import { InjectedDialog } from '@masknet/shared'
+import { WalletStatusBox } from '../../../components/shared/WalletStatusBox'
+import GameList from './GameList'
+import GameWindow from './GameWindow'
+import GameShareDialog from './GameShareDialog'
+
+import { WalletMessages } from '../../Wallet/messages'
+import type { GameInfo, GameNFT } from '../types'
+import { useI18N } from '../../../utils'
+
+const WalletConnectDialog = () => {
+    const { t } = useI18N()
+    const [isGameShow, setGameShow] = useState(false)
```

Comment:
Recommend to use `unstated-next` to create a state context and move these states into the context.